src/state/opamFileTools.ml

```diff
@@ -669,7 +669,7 @@ let lint ?check_extra_files ?(check_upstream=false) t =
             if not_corresponding = [] then None
             else
             let msg =
-              Printf.sprintf "Cheksum%s %s don't verify archive"
+              Printf.sprintf "Checksum%s %s don't verify archive"
```

Comment:
"doesn't/don't match the archive" ?
Or better: "The archive doesn't match checksum(s) xxx"